### PR TITLE
Add /plot visit [player] command

### PR DESCRIPTION
**Pull request by tyler569**
This adds the ability to visit a player's claimed plot.

To accomplish this, I add a new function to `database` that can find a
plot's x, z coordinates by name, a new option to handle_plot_command,
and new entries to the completion tree.

I tested this to work for multiple players, and it does not break if
someone has multiple plots, though you will always only go to one.

### CODE REVIEW

**Comment by AL1L** [comment]
How does it handle if a player has more than one plot? It just get the first one it finds?

**Review by AL1L — APPROVED** [review]
Well... it works

File: src/plot/commands.rs [hunk 1]
```diff
@@ -616,14 +625,38 @@ lazy_static! {
                 name: Some("a"),
                 parser: None,
             },
-            // 40: /p auto
+            // 40: /p middle
```
Comment (AL1L) [inline src/plot/commands.rs:632]: Out of scope ;) I'll let it slide haha

File: src/plot/commands.rs [hunk 2]
```diff
@@ -51,6 +51,15 @@ impl Plot {
                 let center = Plot::get_center(plot_x, plot_z);
                 self.players[player].teleport(center.0, 64.0, center.1);
             }
+            "visit" | "v" => {
+                let plot = database::get_owned_plot(args[0]);
```
Comment (AL1L) [inline src/plot/commands.rs:55]: Can you validate that they put enough args?

Comment (AL1L) [inline src/plot/commands.rs:59]: Can you validate that they put enough args?

**Review by AL1L — CHANGES_REQUESTED** [review]

**Comment by tyler569** [comment]
Currently, if a player has more than one plot `/p v player` will take you to the first one SQLite returns, likely the first one they claimed. We could make it any with `/p v player [number]` or something and some more complicated SQL, but I see that more as an improvement for later than super necessary. I can work on it if people feel differently though.

I've updated the branch to check argument count.

**Review by StackDoubleFlow — APPROVED** [review]

**Pull request merged**